### ISSUE TITLE
display `vscale` and `epslevel` rather than `Vscale` and `Epslevel` 

### ISSUE THREAD

**Issue by trefethen** [chebfun/chebfun#1445]
In Chebfun, our convention is that the correct spelling of `epslevel`, `vscale`, and `hscale` is all lower case.  However, when we display  a chebfun, they come out with initial upper-case:
```
>> x = chebfun('x')
x =
   chebfun column (1 smooth piece)
       interval       length   endpoint values  
[      -1,       1]        2        -1        1 
Epslevel = 1.110223e-15.  Vscale = 1.
```
This happens though even if you try to access such variables, it won't work:
```
>> x.Epslevel
Error using chebfun/get (line 180)
'Epslevel' is not a recognized CHEBFUN property name.
Error in chebfun/subsref (line 144)
        out = get(f, idx); 
>> x.Vscale
Error using chebfun/get (line 180)
'Vscale' is not a recognized CHEBFUN property name.
Error in chebfun/subsref (line 144)
        out = get(f, idx); 
```
I think the output should be simply `epslevel` and `vscale`.  I've taken the liberty of assigning this issue to our style-wallah, @mhsnjvd.  If nobody objects, Mohsin, would you be willing to make this change?

**Comment by Hadrien-Montanelli** [comment]
In that case, would it be possible to get rid of the full stop after the displayed value? 

Hadrien

**Comment by trefethen** [comment]
Yes please!  I had not dared to ask for so much :) .

**Comment by aaustin141** [comment]
Closed in #1446.
